doc/presentation/slides.md

```diff
@@ -72,20 +74,48 @@
 
   ![](./system-overview.svg)
 
+- Orange is our code.
+
 ---
 
 # System Overview
 
-- This is what it will like:
+- This is the grand plan:
 
   ![](./system-overview-future.svg)
 
+- Orange is our code.
+
 --
 
 - Let's dig into the different pieces.
 
 ---
 
+# Rust
+
+- Picked because of a [blog post](https://dev.to/lcsfelix/using-rust-blurz-to-read-from-a-ble-device-gmb) that I found.
+
+- Rust is probably not the **best** language for this.
+
+  - Bluetooth stack on linux is quite dynamic in places.
+
+  - Cross-compiling with `cross` is okay to set up, but a bit slow.
+
+  - We found a [Python project](https://github.com/JsBergbau/MiTemperature2) halfway through, doing the same.
+
+- It was fun anyway:
+
+  - Good chance to work on something together during lockdown.
+
+  - We're both starting to use Rust for work, so good for learning.
+
+        - Andrew is working on [crosvm](https://chromium.googlesource.com/chromiumos/platform/crosvm/) at the moment.
+
+        - [FutureNHS](https://github.com/FutureNHS/futurenhs-platform/) is using Rust on the backend.
```

Comment:
annoyingly that's how much indentation you need for remark to render it as a doubly-indented point. It might be possible to remove some newlines so that github doesn't render it as a code block. I will give it a go.

Reply:
Added a prettier-ignore block with a note about https://github.com/prettier/prettier/issues/5019